apigee-test-shared/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "apigee-shared-test",
+  "version": "0.0.1",
+  "description": "Used to test apigee proxies",
+  "main": "index.js",
+  "dependencies": {},
+  "devDependencies": {
+    "chai": "4.2.0",
+    "mocha": "5.2.0",
+    "sinon": "7.2.3"
+  },
+  "scripts": {
+    "test": "mocha --recursive tests"
```

Comment:
this script has been changed on master: https://github.com/wpengine/CIAM-OAuth2/blob/master/package.json#L14